tests/component/test_watchdog.py

```diff
@@ -0,0 +1,81 @@
+from nidaqmx import Task
+from nidaqmx.constants import WatchdogAOExpirState, WatchdogCOExpirState
+from nidaqmx.system import Device
+from nidaqmx.system.watchdog import AOExpirationState, COExpirationState
+
+
+def test___call_cfg_watchdog_ao_expir_states___no_error(
+    generate_watchdog_task: Task, sim_9189_device: Device, sim_9263_device: Device
```

Comment:
`generate_watchdog_task` is a function that returns a `Task`. It isn't a `Task`. Use `Callable[[], Task]`

Reply:
Mypy probably didn't flag this because nidaqmx doesn't have a py.typed file yet and we are using ignore_missing_imports. Maybe there's a better way to configure mypy for this situation, but it's not important to figure it out now.

Reply:
Changed